WBooks/Views/Login/LoginViewController.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  LoginViewController.swift
+//  WBooks
+//
+//  Created by Guido Marucci Blas on 4/3/16.
+//  Copyright © 2016 Wolox. All rights reserved.
+//
+
+import UIKit
+import Core
+import GoogleSignIn
+
+final class LoginViewController: UIViewController, GIDSignInUIDelegate {
+
+    fileprivate lazy var _myView: LoginView = LoginView.loadFromNib()!
+    
+    override func loadView() {
+        view = _myView
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        //Sets the Login and LogOut buttons
+        _myView.loginButton.addTarget(self, action: #selector(loginClick(sender:)), for: .touchUpInside)
+        _myView.logOutButton.addTarget(self, action: #selector(logOutClick(sender:)), for: .touchUpInside)
+        _myView.statusText.textAlignment = NSTextAlignment.center
+         
+        GIDSignIn.sharedInstance().uiDelegate = self
+        
+        // Automatically sign in the user.
+        GIDSignIn.sharedInstance().signInSilently()
+        
+        // [START_EXCLUDE]
+        NotificationCenter.default.addObserver(self,
+                                               selector: #selector(receiveToggleAuthUINotification(_:)),
+                                               name: NSNotification.Name(rawValue: "ToggleAuthUINotification"),
+                                               object: nil)
+        _myView.statusText.text = "Initialized Swift app..."
+        toggleAuthUI()
+        // [END_EXCLUDE]
+    }
+    
+    @objc
+    func loginClick(sender: UIButton) {
+        print("Login")
+        GIDSignIn.sharedInstance().signIn()
+    }
+    
+    // [START signout_tapped]
+    @objc
+    func logOutClick(sender: AnyObject) {
+        GIDSignIn.sharedInstance().signOut()
+        // [START_EXCLUDE silent]
+        _myView.statusText.text = "Signed out."
+        toggleAuthUI()
+        // [END_EXCLUDE]
+    }
+  
+    // [START toggle_auth]
+    //Defines which button has be to shown, depending if the user has to log in, of if it's already logged.
+    func toggleAuthUI() {
+        if GIDSignIn.sharedInstance().hasAuthInKeychain() {
+            // Signed in
+            _myView.loginButton.isHidden = true
+            _myView.logOutButton.isHidden = false
+        } else {
+            _myView.loginButton.isHidden = false
+            _myView.logOutButton.isHidden = true
```

Comment:
This could be done shorter with:
```
_myView.logInButton.isHidden = GIDSignIn.sharedInstance().hasAuthInKeychain()
_myView.logOutButton.isHidden = !GIDSignIn.sharedInstance().hasAuthInKeychain()
```
And then use the if for the `statusText` if it should only be used for one case. If it should be for both, you could do something like: `_myView.statusText.text = GIDSignIn.sharedInstance().hasAuthInKeychain() ? "Already logged in (or whatever)" : "Google Sign in\niOS Demo"`